eZ/Publish/Core/Repository/Tests/Service/Mock/Base.php

```diff
@@ -55,6 +57,9 @@ abstract class Base extends TestCase
     /** @var \PHPUnit\Framework\MockObject\MockObject|\eZ\Publish\Core\Repository\Helper\ContentTypeDomainMapper */
     private $contentTypeDomainMapperMock;
 
+    /** @var \PHPUnit\Framework\MockObject\MockObject|\eZ\Publish\Core\Repository\Permission\LimitationService */
+    protected $limitationServiceMock;
```

Comment:
Why not `private`?

Reply:
> Why not `private`?

No reason, good catch. Fixed and squashed into 5b1b20916c.